shared/styles/style-guide.desktop.js

```diff
@@ -44,7 +44,8 @@ export const globalColors = {
 }
 
 export const globalResizing = {
-  login: {width: 700, height: 580},
+  // Add 25px to login for the OS menubar
+  login: {width: 700, height: 605},
```

Comment:
I think the reason he changed it was to account for the top window bar thing with the close/minimize icons. but `setContentSize` should account for that without us having to add the 25px.